src/microbe_stage/ColonyCompoundBag.cs

```diff
@@ -120,4 +135,9 @@ private IEnumerable<CompoundBag> GetCompoundBags()
     {
         return Colony.ColonyMembers.Select(p => p.Compounds);
     }
+
+    private bool IsNoneUseful(Compound compound)
```

Comment:
Usually programming conditions are easier to understand when they are positive, ie. this method should be the opposite (`IsAnyUseful`).

Reply:
Though as this also is used to check if *any* colony member finds this useful, I'd actually want this to be named `IsUsefulToAnyColonyMember` or maybe `IsUsefulInAnyCompoundBag`.